src/main/kotlin/utils/request.kt

```diff
@@ -0,0 +1,13 @@
+package utils
+
+import com.google.gson.Gson
+import okhttp3.OkHttpClient
+import okhttp3.Request
+
+@Suppress("BlockingMethodInNonBlockingContext")
+inline fun <reified T> request(token: String, url: String): T {
```

Comment:
This should be renamed to something like github request or token request, because normally a request doesn't use an Authorization header